defaults/main.yml

```diff
@@ -27,6 +27,28 @@ gitlab_version: '8.16'
 # GitLab will be configured with HTTPS enabled by default
 gitlab_domain: [ 'code.{{ ansible_domain }}' ]
 
+# .. envvar:: gitlab_pages_domain
+#
+# The main domain served by Gitlab Pages.
+#
+# This should be a different one than ``gitlab_domain`` to prevent cross domain cookie attacks.
+gitlab_pages_domain: ''
+
+# .. envvar:: gitlab_pages_domain
+#
+# List of domains Gitlab Pages should serve.
+#
+# They have to be ``gitlab_domain`` subdomains and respect the formato of the `` ``subject_alt_names``
+# variable in the ``debops.pki`` role.
+#
+# Ex. `[ 'dns:host1.example.org', 'DNS:host2.sub1.example.com' ]``
+gitlab_pages_alt_names: []
```

Comment:
I don't like this at all, but Let's encrypt doesn't support wildcard domains yet, and we are forced to list all the possible users' domain beforehand.

Also, the prepended `DNS:` is ugly.

Any hints?